### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andrum993**
Highlight EOL warning, tweak grammar.

### CODE REVIEW

**Comment by andrum993** [comment]
If my suggested IMPORTANT callout seems too strong, there's also an NOTE version - see https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

File: README.md [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
-
-To learn more, follow the [Getting Started with Raspberry Pi Pico-series](https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf) tutorial.
-
-
-
-
+> [!IMPORTANT]
+> We are not intending to support the pico-setup-windows installer in future; we're instead recommending people use our Visual Studio Code extension available at https://marketplace.visualstudio.com/items?itemName=raspberry-pi.raspberry-pi-pico. This has been developed to better integrate across all platforms (MacOS, Linux and Windows).
+>
+> To learn more, follow the [Getting Started with Raspberry Pi Pico-series](https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf) tutorial.
```
Comment (nathan-contino) [inline README.md:6]:
```suggestion
> [!IMPORTANT]
> In the future, the pico-setup-windows installer will not be supported. Instead, use the official Visual Studio Code extension available at https://marketplace.visualstudio.com/items?itemName=raspberry-pi.raspberry-pi-pico. This extension handles setup on multiple platforms, including Linux, macOS, and Windows.
>
> To learn more, follow the [Getting Started with Raspberry Pi Pico-series](https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf) tutorial.

```

**Comment by nathan-contino** [comment]
Thanks for this contribution. I made some additional wording improvements; if you accept the suggestion, I'll merge this in.

**Comment by andrum993** [comment]
Done 👍

**Pull request merged**